node-launchpad/src/components/help.rs

```diff
@@ -0,0 +1,256 @@
+        })
+    }
+}
+
+impl Component for Help {
+    fn init(&mut self, _area: Rect) -> Result<()> {
+        Ok(())
+    }
+
+    fn draw(&mut self, f: &mut Frame<'_>, area: Rect) -> Result<()> {
+        if !self.active {
+            return Ok(());
+        }
+        // We define a layout, top and down box.
+        let layout = Layout::default()
+            .direction(Direction::Vertical)
+            .constraints(vec![
+                Constraint::Length(1),
+                Constraint::Min(7),
+                Constraint::Max(9),
+            ])
+            .split(area);
+
+        // ==== Header =====
+        let header = Header::new();
+        f.render_stateful_widget(header, layout[0], &mut SelectedMenuItem::Help);
+
+        // ---- Get Help & Support ----
+        // Links
+
+        let quickstart_guide_link = Hyperlink::new(
+            "docs.autonomi.com/getstarted",
+            "https://docs.autonomi.com/getstarted",
+        );
+        let beta_rewards_link = Hyperlink::new("autonomi.com/beta", "https://autonomi.com/beta");
+        let get_direct_support_link =
+            Hyperlink::new("autonomi.com/support", "https://autonomi.com/support");
+        let download_latest_link =
+            Hyperlink::new("autonomi.com/downloads", "https://autonomi.com/downloads");
+
+        // Content
+        let rows_help_and_support = vec![
+            Row::new(vec![
+                Cell::from(Line::from(vec![Span::styled(
+                    "See the quick start guides:",
+                    Style::default().fg(GHOST_WHITE),
+                )])),
+                Cell::from(Line::from(vec![Span::styled(
+                    "To join the Beta Rewards Program:",
+                    Style::default().fg(GHOST_WHITE),
+                )])),
+            ]),
+            Row::new(vec![
+                Cell::from(
+                    quickstart_guide_link
+                        .to_string()
+                        .into_text()
+                        .unwrap()
+                        .clone(),
+                ),
+                Cell::from(beta_rewards_link.to_string().into_text().unwrap().clone()),
+            ]),
+            Row::new(vec![
+                // Empty row for padding
+                Cell::from(Span::raw(" ")),
+                Cell::from(Span::raw(" ")),
+            ]),
+            Row::new(vec![
+                Cell::from(Line::from(vec![Span::styled(
+                    "Get Direct Support:",
+                    Style::default().fg(GHOST_WHITE),
+                )])),
+                Cell::from(Line::from(vec![Span::styled(
+                    "Download the latest launchpad:",
+                    Style::default().fg(GHOST_WHITE),
+                )])),
+            ]),
+            Row::new(vec![
+                Cell::from(
+                    get_direct_support_link
+                        .to_string()
+                        .into_text()
+                        .unwrap()
```

Comment:
Can we use `eyre` here as well, and also in the ones below.

Reply:
For the links, I believe than an `unwrap()` here will not be an `Error` as we are building the hyperlink in the code and we test.
`.into_text()` on this case, is part of `ansi-to-tui`. Maybe we can handle this when we handle `Hyperlink`.
What do you think?